components/analytics/team_analytics/team_analytics.tsx

```diff
@@ -145,8 +153,8 @@ export default class TeamAnalytics extends React.PureComponent {  //? do I have
         }
 
         const stats = this.props.stats[this.state.team.id];
-        const postCountsDay = formatPostsPerDayData(stats[StatTypes.POST_PER_DAY]);
-        const userCountsWithPostsDay = formatUsersWithPostsPerDayData(stats[StatTypes.USERS_WITH_POSTS_PER_DAY]);
+        const postCountsDay = formatPostsPerDayData(stats[StatTypes.POST_PER_DAY], stats); //! need to sort out the arguments for this function
+        const userCountsWithPostsDay = formatUsersWithPostsPerDayData(stats[StatTypes.USERS_WITH_POSTS_PER_DAY], stats);
```

Comment:
This is one of the instances where I'm a little confused with which parameter should be set as the second argument.  I believe it's currently defined as:
`formatUsersWithPostsPerDayData(labels: string[], data: any)`
`data` as type `any` is ambiguous for me presently

Reply:
So, we shouldn't be adding additional arguments to functions in a TypeScript migration, as this will change the behaviour of the component.

If the function expects a second argument, you can pass in `undefined`.
If the function expects the second argument to be an actual object, let me know as that means our type definitions will need to be updated.